src/mmw/js/src/analyze/templates/pointSourceTable.html

```diff
@@ -37,10 +37,23 @@
   </table>
 </div>
 <div class="row paging-ctl-row">
-  <button class="btn-prev-page btn btn-primary">
-    <span aria-hidden="true">&larr;</span>
-  </button>
-  <button class="btn-next-page btn btn-primary">
-    <span aria-hidden="true">&rarr;</span>
-  </button>
+    <p>
+        {% if totalPages %}
+            {{currentPage}} of {{totalPages}}
+        {% endif %}
+    </p>
+    {% if hasPreviousPage %}
+        <button class="btn-prev-page btn btn-primary"
+          data-toggle="tooltip" title="Previous Page"
+          data-placement="right">
+            <span aria-hidden="true">&larr;</span>
+        </button>
+    {% endif %}
+    {% if hasNextPage %}
+    <button class="btn-next-page btn btn-primary"
```

Comment:
This `if` block should probably maintain a level of indentation like the `if` block above it.  Same for the water quality template.